screens/profiles/profiles.gui_script

```diff
@@ -45,8 +45,6 @@ end
 
 local function activate(slot)
 	profiles.set_active(slot)
-	storage.load(profiles.get_active())
-	trainer.load(profiles.get_active())
 	dex.load(profiles.get_active())
 	network.load_profile(profiles.get_active())
```

Comment:
Same here, I would love if we could remove these load calls. Really they should be triggered as a result of `set_active` in some way, and then `set_active_complete` can go away.